pennylane/devices/tests/test_measurements.py

```diff
@@ -139,8 +141,7 @@ def test_tensor_observables_can_be_implemented(self, device_kwargs):
 
         @qml.qnode(dev)
         def circuit():
-            if dev.supports_operation(qml.PauliX):  # ionq can't have empty circuits
-                qml.PauliX(0)
+            qml.PauliX(0)
```

Comment:
Do we expect this departure from checking if X is supported to cause pain with any externals?